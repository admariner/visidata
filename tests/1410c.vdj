#!vd -p
{"sheet": "global", "col": null, "row": "numeric_binning", "longname": "set-option", "input": "True", "keystrokes": "", "comment": null}
{"longname": "open-file", "input": "sample_data/y77d-th95.json.gz", "keystrokes": "o"}
{"sheet": "y77d-th95", "col": "geolocation", "row": "", "longname": "type-len", "input": "", "keystrokes": "z#", "comment": "set type of current column to len"}
{"sheet": "y77d-th95", "col": "geolocation", "row": "", "longname": "freq-col", "input": "", "keystrokes": "Shift+F", "comment": "open Frequency Table grouped on current column, with aggregations of other columns"}
