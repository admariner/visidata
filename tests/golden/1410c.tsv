geolocation	count	percent	histogram
2	988	98.80	■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■
	12	1.20	
